### ISSUE TITLE
Source for MKS Robin Nano v1.2 TFT35 - Mks_Nano35_V2.0.3.6 Release file.rar 

### ISSUE THREAD

**Issue by Tomza86** [makerbase-mks/MKS-Robin-Nano-V1.X#106]
Hi, 

I'm using this firmware on my 3D printer: Mks_Nano35_V2.0.3.6 Release file.rar. Great job by the way. 
Is it possible to get this firmware with all that settings as source files (I want to change some features)?  
I just want to add some extra settings and check why some of the current settings won't work correctly. I have PWC module v3.0 and when the printer ends then is only the message "Closing Machine....". The printer won't shut down like newer. 
The features I want to add: 
- extra driver on the motherboard and use Z Alignment function. Separated drivers for the double Z axis.
- use the Pause feature (M600) and the advanced park feature. 
- Mech bed leveling, I'm using bltouch and instead of endstop on Z-axis and if I want to use autobed leveling then only Z offset appears on the screen. 

Thanks

**Comment by WashingtonJunior** [comment]
Yeah, this firmware is way better than the open source version. Indeed, MKS could release this code to us, so we can set some new features.
